core/ui/src/components/domains/EditExternalDomain.vue

```diff
@@ -0,0 +1,230 @@
+<!--
```

Comment:
Can we rename this component to `EditExternalDomainModal` for uniformity with other modals?